### PR TITLE
Django 3.2 & Python 3.9 support

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -23,6 +24,7 @@ deps =
     django22: Django>=2.2.1,<2.3
     django30: Django>=3.0,<3.1
     django31: Django>=3.1,<3.2
+    django31: Django>=3.2rc1,<3.1
```
Comment (daviddavis) [inline tox.ini:27]: Will update this once django 3.2 releases.

```suggestion
    django31: Django>=3.2,<3.1
```

File: tox.ini [hunk 2]
```diff
@@ -23,6 +24,7 @@ deps =
     django22: Django>=2.2.1,<2.3
     django30: Django>=3.0,<3.1
     django31: Django>=3.1,<3.2
+    django32: Django>=3.2rc1,<3.1
```
Comment (daviddavis) [inline tox.ini:27]: Will drop the `rc1` here when django 3.2 is out.

**Comment by N1K1TAS95** [comment]
...

**Review by dunix15 — APPROVED** [review]

**Comment by N1K1TAS95** [comment]
Is it possible to have this merge also on pip [repository](https://pypi.org/project/django-currentuser/)?

**Comment by daviddavis** [comment]
Ping @belugame :wave:

**Pull request merged**